src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CurrentUserOnly.Windows.cs

```diff
@@ -131,14 +131,51 @@ public NamedPipeTest_CurrentUserOnly_Windows(TestAccountImpersonator testAccount
             _testAccountImpersonator = testAccountImpersonator;
         }
 
+        [OuterLoop]
+        [ConditionalFact(nameof(IsAdminOnSupportedWindowsVersions))]
+        public async Task Connection_UnderDifferentUsers_CurrentUserOnlyOnServer_DoesNotBlockOthers()
```

Comment:
```suggestion
        public async Task Connection_UnderDifferentUsers_CurrentUserOnlyOnServer_FailingClientConnectionAttemptsDoNotBlockSuccessfulClientConnections()
```
?

Reply:
The last "connection" is redundant with the first one.
```suggestion
        public async Task Connection_UnderDifferentUsers_CurrentUserOnlyOnServer_InvalidClientConnectionAttempts_DoNotBlockSuccessfulClient()
```

Reply:
yup